sys/include/div.h

```diff
@@ -75,6 +100,20 @@ static inline uint64_t div_u64_by_15625(uint64_t val)
 }
 
 /**
+ * @brief Integer divide val by 625, 64 bit version
+ *
+ * @param[in]   val     dividend
+ * @return      (val / 625)
+ */
+static inline uint64_t div_u64_by_625(uint64_t val)
+{
+    if (val > 10485759996ull) {
```

Comment:
and further this number is used at least twice here, so a macro might also help to reduce copy-paste errors